lib/asset_audit.rb

```diff
@@ -0,0 +1,104 @@
+class AssetAudit
+  def cache_bust
+    "cache_bust=#{rand(1e8)}"
+  end
+
+  def self.dump
+    whitehall_base_url = Plek.new.external_url_for('assets-origin')
+    asset_manager_base_url = Plek.new.external_url_for('draft-assets')
+
+    CSV($stdout, headers: %i[whitehall asset_manager], write_headers: true) do |csv|
+      AttachmentData.find_each do |attachment_data|
+        visible = attachment_data.visible_to?(nil)
```

Comment:
Might be worth putting some time into filtering invisible attachment data in SQL.  I had a bit of a look and couldn't find a quick solution, though.

Reply:
We think this would be nice to have, but not really worth the effort.